test/unit/document_test.rb

```diff
@@ -152,4 +152,17 @@ class DocumentTest < ActiveSupport::TestCase
 
     refute draft.document.similar_slug_exists?
   end
+
+  test "current government can only be set once" do
+    initial_government = FactoryGirl.create(:government, name: "2004 to 2009 Labour government", start_date: '2005-05-06', end_date: '2010-05-11')
+    document = create(:document)
+    document.ensure_document_has_government
+
+    assert_equal initial_government, document.government
+    Timecop.freeze('2012-05-12') do
```

Comment:
Since the code doesn't use any dates to pick which `Government` record to assign to, this seems a little misleading